src/dev/flang/tools/Fuzion.java

```diff
@@ -690,8 +692,16 @@ private Runnable parseArgsAceMode(String[] args)
    */
   private Runnable parseArgsForBackend(String[] args)
   {
+    var argsLeft = args;
+
     for (var a : args)
       {
+        argsLeft = Arrays.copyOfRange(argsLeft, 1, argsLeft.length);
```

Comment:
This causes quadratic performance: for n arguments, you are copying n, n-1, n-2, etc. to a new array argsLeft. 
Quadratic performance must be avoided, this will very likely cause performance problems later.